src/components/Progress/Progress.tsx

```diff
@@ -0,0 +1,233 @@
+import React, {Component} from 'react';
+import {block} from '../utils/cn';
+import _ from 'lodash';
```

Comment:
Use separate methods (`import _get from 'lodash/get'`) to avoid bundling of full lodash